GoogleForCreators/web-stories-wp
# Template Updates: Wellbeing

## Context

Part of 1.7 release is updating existing templates to avoid custom animations and unnecessary prepublish checklist suggestions.

## Summary

Wellbeing Template revisions 
- remove custom animations and replace them with effects available in editor 
- add arrows and cta circle as stickers
- adjust blob shapes as directed 

## Relevant Technical Choices

Used my own judgement for what direction everything should fly in. 

## To-do


## User-facing changes

See figma for revised template: https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories-Post-Stable?node-id=7357%3A546501
List of changes can be found here: https://github.com/google/web-stories-wp/issues/6976

**Before**

https://user-images.githubusercontent.com/10720454/114105754-39edfa80-9882-11eb-8288-a355157f51ef.mp4


**After**

https://user-images.githubusercontent.com/10720454/114104162-3442e580-987f-11eb-8363-dcbaebe41657.mp4



## Testing Instructions

Use the wellbeing template from dashboard/explore templates and see that the template has been updated in line with Figma. There should be no more custom animations and PPC suggestion about clickable link element size should be resolved on page 4.

### QA






This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?

No

### Does this PR change what data or activity we track or use?

No

### Does this PR have a legal-related impact?

No

## Checklist




---



Fixes #6976
